docs/guides/ide/overview.md

```diff
@@ -1,54 +1,106 @@
+
+![Source Control](../../images/guides/ideSourceControl.png ':size=900')
+
+### Debugging
+
+You may debug RPG (including RPGLE and OPM RPG), COBOL, C/C++, and CL programs using the debug views in VS Code. The **IBM i Debug** extension supports batch debugging and service entry points. For information on debug features, go to the [Debugging](./guides/ide/debug.md) page.
+
+![Debugging](../../images/guides/debugging.png ':size=800')
+
+### Extensions
+
+When a new workspace is created, several IBM i related extensions are installed to help you with development such as the **IBM i Developer**, **Code for IBM i**, **IBM i Project Explorer**, and much more. For information on all installed extensions, go to the [Extensions](./guides/ide/extensions.md) page.
+
+![Extensions View](../../images/guides/extensionsView.png ':size=500')
+
+### Panel
+
+The panel is used to hold additional views below the editor. You may re-arrange views between different view containers by dragging the view's heading.
+
+* **Problems** - View informational, warning, and error diagnostics.
+* **Output** - View output channels for logging of built in VS Code features and various extension.
+* **Debug Console** - Shows debugging output.
+* **Terminal** - Integrated terminal that starts at the root of the workspace.
+* **Job Log** - Visuzlize IBM i job logs after running a build or compile.
+
+![Panel](../../images/guides/ideWorkspacePanel.png ':size=900')
+
+Note: To create a new terminal, select **Terminal > New Terminal**
+
+![Terminal](../../images/guides/ideWorkspaceTerminal.png ':size=500')
+
+### Command Palette
+
+To run a command, navigate to the command palette at the top of the page and select the **Show and Run Commnads** option. Selecting this option will prefix the search with an `>` which will then show all commands which can be run. This can also be brought up using **View > Command Palette...**.
+
+![Commands](../../images/guides/ideWorkspaceCommand.png ':size=700')
+
+### Settings
+
+Settings can be used to manage extension settings, keyboard shortcuts, themes (colors, file icons, and product icosn), and much more. To change any of these settings, click the **Manage** gear icon at the bottom left and select the appriopriate option. For information on settings, go to the [Settings](./guides/ide/settings.md) page.
+
+![Settings](../../images/guides/ideWorkspaceSettings.png ':size=350')
 
+To return to the dashboard, click the white **IBM i Developer Workspaces Web** button in the bottom left status bar and use the **IBM i Developer Workspaces: Open Dashboard** command.
 
+![Return to Dashboard](../../images/guides/ideWorkspaceReturn.png ':size=800')
 
 For information on using ARCAD tools, see [How to use ARCAD integration with IBM i Modernization Engine for Lifecycle Integration](https://supportcontent.ibm.com/support/pages/how-use-arcad-integration-ibm-i-modernization-engine-lifecycle-integration-merlin)
```

Comment:
@eric-simpson Almost all the IDE pages have this same exact text on every page. Can I remove this and just add a page that briefly explains the ARCAD tools and links to their docs site? Also the content on this support page is outdated.

Reply:
this will be updated later across all the pages